spec/services/calendar_service_spec.rb

```diff
@@ -1,8 +1,10 @@
-RSpec.describe CalendarService do
+RSpec.describe CalendarService, skip_seed: true do
+  let(:organization) { create(:organization, skip_items: true) }
+
   describe ".calendar" do
     it "should print the calendar correctly" do
       storage_location = create(:storage_location,
-        time_zone: "America/New_York")
+        time_zone: "America/New_York", organization: organization)
```

Comment:
This fixes it, but I don't really understand this test. Unsure if this storage location should be associated with the organization. Should other stuff also be associated with the organization? Like:

```rb
        partner1 = create(:partner, name: "Partner 1")
        partner2 = create(:partner, name: "Partner 2")
```

Adding `organization: organization` does not have an impact on the test outcome but I don't understand the test.

Reply:
If you look in the `partner` factory, it'll automatically associate itself with the last organization in the DB if it's not given one. We haven't changed that behavior.